hbw/app/javascript/packs/hbw/components/app.jsx

```diff
@@ -27,13 +30,21 @@ const App = ({ Button, TaskList, Forms }) => {
     };
   }, []);
 
-  return (
-    <>
+  if (connectionError.exist && formsContainer) {
+    return (
+      <>
+        {createPortal(<HBWError errorHeader={connectionError.header} error={connectionError.body} />, formsContainer)}
+      </>
+    );
+  } else {
```

Comment:
Not here. If you want to render an error in the forms portal do it in the `<Container />`. 

On the other hand, we will have silent nothingness if a page has a button portal only (e.g. HOMS main page, or most of the HOPER  pages).
All these components are quite independent so we should or render errors inside every portal either render _some general_ error, not inside any portal, like a pop-up.

Reply:
This was discussed, pop-up was rejected. 
Perhaps it is worth changing the display of the number of tasks in the widget menu in the form of question marks if the kamunda is unavailable. 
It might be worth discussing this with the designer.